hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/HazelcastWriters.java

```diff
@@ -162,7 +301,8 @@ private static SerializableClientConfig serializableConfig(ClientConfig clientCo
             entries = new ArrayList<>();
         }
 
-        @Override @Nonnull
+        @Override
```

Comment:
you don't need to reformat these lines, you can configure intellij to put them on same line.